docs/concepts/sanitization.md

```diff
@@ -0,0 +1,55 @@
+# Sanitization
+
+> I wish it were Santa-ization.  Who wouldn't like that?
+>
+> &#8212; Anonymous
+
+## Definition
+
+[Sanitization](https://en.wikipedia.org/wiki/HTML_sanitization) within the context of Bandicoot refers to cleaning an HTML string to remove potentially malicious content. Unsanitized HTML is vulnerable to [cross site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) which is something you need to avoid in your code. Bandicoot does **NOT** automatically sanitize your HTML, however it provides a way for you to bring your own sanitizing function to the party.
+
+## Why should you care?
+
+After receiving incoming HTML, Bandicoot inserts the content into the DOM via React's [dangerouslySetInnerHTML](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml).
+React calls it dangerous for good reason -- if the HTML string has a `<script>` element or other javascript embedded into the dom elements,
+that javascript will be executed during deserialization. This is a security risk known as
+[cross site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting).
+
+## What this means for you
+
+### Sanitization
+Since Bandicoot does **NOT** automatically sanitize incoming or outgoing HTML, it is up to you as the developer/consumer to make sure that sanitization happens. However, Bandicoot does expose a [sanitizeHTML prop](/components/rich-text-editor.md#props) on the [RichTextEditor component](/components/rich-text-editor.md) which allows you to provide a sanitization function.
+
+If a sanitization function is provided through the `sanitizeHTML` prop, Bandicoot will pass incoming html strings through this function before inserting the content into the DOM (see [note 1](/concepts/sanitization.md#notes)). If provided, Bandicoot will also pass outgoing html through the `sanitizeHTML` function after serialization (see [note 2](/concepts/sanitization.md#notes)). If a `sanitizeHTML` function is not provided, Bandicoot will simply pass through the *unsanitized* html string - which is **NOT RECOMMENDED**!
```

Comment:
html -> HTML

Reply:
`Bandicoot will simply pass through the unsanitized HTML string`

Isn't it more accurate to say `Bandicoot will dangerouslySetInnerHTML with the unsanitized HTML`

Reply:
updated html vs HTML inconsistency.  Also updated the text to include a bit more clarification on the dangerouslySetInnerHTML stuff.